taxcalc/functions.py

```diff
@@ -501,46 +516,31 @@ def StdDed(DSI, _earned, STD, p04470, e00100, e60000,
     if (MARS == 3 or MARS == 6) and (MIDR == 1):
         _standard = 0.
 
-    return (_standard, c04200, _numextra, c15200, c15100, x04500, _txpyers,
-            c04100)
+    return (_standard, c04200, _numextra, c15200, c15100, x04500, c04100)
 
 
 @iterate_jit(nopython=True, puf=False)
-def TaxInc(c00100, c04470, _standard, e37717, c21060, c21040,
-           p04470, c04200, c04500, c04600, x04500,
-           e04805, t04470, f6251, _exact, _feided, c04800, MARS,
+def TaxInc(c00100, _standard, e37717, c21060, c21040, c04500, c04600,
+           e04805, _feided, c04800, MARS,
            II_rt1, II_rt2, II_rt3, II_rt4,
            II_rt5, II_rt6, II_rt7, II_brk1, II_brk2, II_brk3,
-           II_brk4, II_brk5, II_brk6, _numextra, FDED, c02700):
-
-    # c04500 = c00100 - max(c04470, max(c04100, _standard + e37717))
+           II_brk4, II_brk5, II_brk6, c02700):
+    """
+    TaxInc function: ...
+    """
     c04500 = max(0, c00100 - max(c21060 - c21040, _standard + e37717))
 
-    # Check with Dan whether this is right!
-    # if c04470 > _standard:
```

Comment:
Sean said:
> @MattHJensen  The rest of the changes look good to me except this part.
> @martinholmer  Can we still keep part of the comments here? Although we are doing the optimization of itemizing in calculate.py, it would still be helpful to keep use this when performing the SAS-validation. Specifically, can we keep [the following commented out code]:
```
# if c04470 > _standard:
#    _standard = 0.
# if c04470 <= _standard:
#    c04470 = 0.
```

I don't understand what you mean when you say "performing the SAS-validation."  To me validation, means that, when using the same input, two models produce the same output.  If you uncomment these four lines, then you don't have the same logic as the Tax-Calculator, you have some third model.  (If this is not true, please explain why you have to uncomment these lines to do a "validation".)   How can comparing output from a third model with that from the SAS program be considered a SAS-vs-Tax-Calculator validation?  I guess my confusion about this is rooted in the fact that the SAS validation work is all being done in secret, which is contrary to open-source principles.  Nobody, not even those on the core development team, knows how the SAS validation work is being done.  We need much more explanation and transparency in order to understand your request.

cc @MattHJensen @feenberg @GoFroggyRun @Amy-Xu